contracts/vaults/BaseVaults/RibbonThetaVault.sol

```diff
@@ -428,11 +428,16 @@ contract RibbonThetaVault is RibbonVault, RibbonThetaVaultStorage {
     function _startAuction() private {
         GnosisAuction.AuctionDetails memory auctionDetails;
 
-        uint256 currOtokenPremium = currentOtokenPremium;
-
+        address currentOtoken = optionState.currentOption;
+        uint256 currOtokenPremium =
```

Comment:
might make sense to remove redundant calculation in vaultlifecycle. it will probably make up for contract redeployments quickly as its an expensive calculation.

Reply:
Agreed